doc/man1/flux-shell.rst

```diff
@@ -276,7 +276,10 @@ options are supported by the builtin plugins of ``flux-shell``:
   sets ``CUDA_VISIBLE_DEVICES`` to the GPU IDs allocated to the job.
   *OPT* may be set to ``off`` to disable the plugin, or ``per-task``
   to divide allocated GPUs among tasks launched by the shell (sets a
-  different GPU ID or IDs for each launched task)
+  different GPU ID or IDs for each launched task) If *OPT* starts with
```

Comment:
Missing period?

Reply:
Also this refers to the cpu-affinity entry in the man page, but over there, the hwloc_cpuset_t(3) reference is a 404 in the [rendered page](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-shell.html).

Reply:
Oh, nice catch!

Reply:
Hm, that manpage exists on some systems but not on linux,die.net. I guess I'll have to reference the hwloc documentation directly (which won't be too nice in a manpage but oh well)

Reply:
Ok, I pushed an extra commit to fix the missing `hwloc_cpuset_t(3)` link. Used `hwlocality_bitmask(3)` instead, which has the actual `*sscanf(3)` functions used in the shell plugin. Since that manpage also does not exist on linux.die.net, an explicit link directly to the latest hwloc documentation is used.